### ISSUE TITLE
JSON required for tests but not declared

### ISSUE THREAD

**Issue by szabgab** [robmueller/cache-fastmmap#20]
```
Could not load serialization package: JSON : Can't locate JSON.pm in @INC (you may need to install the JSON module) (@INC contains: /__w/cache-fastmmap/cache-fastmmap/blib/lib /__w/cache-fastmmap/cache-fastmmap/blib/arch /usr/local/lib/perl5/site_perl/5.30.3/x86_64-linux-gnu /usr/local/lib/perl5/site_perl/5.30.3 /usr/local/lib/perl5/vendor_perl/5.30.3/x86_64-linux-gnu /usr/local/lib/perl5/vendor_perl/5.30.3 /usr/local/lib/perl5/5.30.3/x86_64-linux-gnu /usr/local/lib/perl5/5.30.3 .) at (eval 13) line 1.
# Looks like your test exited with 2 just after 2.
```
See full output: https://github.com/szabgab/cache-fastmmap/runs/1374034225?check_suite_focus=true

One solution would be to skip that specific test if JSON is not available.

**Comment by szabgab** [comment]
Same with `Sereal::Encoder`

The next run:
https://github.com/szabgab/cache-fastmmap/runs/1374060507?check_suite_focus=true

**Comment by robmueller** [comment]
Have fixed in 1.51 and released to cpan along with your other PR's. Thanks

**Comment by szabgab** [comment]
Nice.
